README.md

```diff
@@ -1 +1,45 @@
-# charmed-superset-rock
\ No newline at end of file
+# Charmed Apache Superset ROCK
+
+This repository contains the packaging metadata for creating a Charmed Superset ROCK. This ROCK image is based on the upstream [Apache Superset](https://downloads.apache.org/superset/) image.
+
+For more information on ROCKs, visit the [rockcraft Github](https://github.com/canonical/rockcraft).
+
+## Building the ROCK
+The steps outlined below are based on the assumption that you are building the ROCK with the latest LTS of Ubuntu.  
+If you are using another version of Ubuntu or another operating system, the process may be different. 
+To avoid any issue with other operating systems you can simply build the image with [multipass](https://multipass.run/):
```

Comment:
Sounds great but doesn't work. the `rockcraft pack` step fails on multipass. Please test end to end on multipass.

Reply:
This is due to firewall rule problems when using lxd snap and docker snap. This has been resolved by initialising lxd before snap installing docker, please have another go.